src/App.tsx

```diff
@@ -1,11 +1,17 @@
 import React from 'react';
 import Mainform from './components/MainForm/Mainform';
+import Countries from './components/Countries/Countries';
+import Splash from './components/LoadingScreen/Splash';
+import Confirmation from './components/FinalPage/Confirmation';
 import './App.css';
 
 function App() {
   return (
     <div className="App">
+    {/* <Countries/> */}
      <Mainform/>
```

Comment:
Mainform is meaningless name which represents nothing!!
use proper components names